docs/blog/newsletter-december-2019.rst

```diff
@@ -0,0 +1,127 @@
+
+* Mac:
+   * Command + Backquote: switch between windows in one app (eg different browser windows).
+   * Command + Tab: switch to next most recently used app. Command + Shift + Tab to change in the opposite direction; hold Command then hit Tab to cycle through open applications.
+   * Command + ~ (tilde): cycle through open windows in the current application.
+   * Option + click green button in a window: maximize the window.
+* Windows:
+   * Alt + Tab: switch to next most recently used app. Alt + Shift + Tab to change in the opposite direction; hold Alt then hit Tab to cycle through open applications.
+   * Alt + D: go to the URL bar (in web browsers).
+   * Ctrl + Shift + Esc: open Task Manager.
+   * Windows key: open the Start menu with the cursor in the Search field.
+   * Windows key + arrows: move application windows on the screen.
+   * Windows key + D (or M): hide all applications, show the desktop.
+   * Windows key + L: lock the screen.
+   * Windows key + P: open a Project panel that lets you mirror your screen and extend it to other displays.
+   * Windows key + X: open a "god mode" panel with links to common administrative controls like Power Options and Settings.
+   * Windows key + Tab: open workspaces preview
+   * F11: toggle full screen.
+
+We also had one suggestion for Linux: Alt + Backquote to switch between windows in a single app (eg different browser windows).
+
+One more tip: if you regularly use several operating systems, consider setting up keyboard mapping so your shortcuts stay the same across systems.
+
+----------------------------------------
+Should documentation have a formal tone?
+----------------------------------------
+
+Is there a requirement for formal tone in documentation? Or is it better for docs be informal? We had a lively discussion on the topic this month.
+
+Our documentarians were mostly of the 'it depends' school of thought here - as so often, there are no hard and fast rules. In particular, everyone agreed that the choice depends a lot on what your audience expects. It's important that your tone should not be jarring, causing it to get in the way of your message.
+
+What "informal" means can vary; going for a conversational style doesn't require being irreverent. You can be professional and conversational at the same time, just as you might be when talking to colleagues or customers in person. Sometimes you'll find that less-formal third-party material is more popular than the formal material written by the company itself. It's possible that its more casual tone makes it more approachable.
+
+On the side of formality, localization is a key factor to consider. Being too informal can make translation challenging, even more so if you're using humour. The community was much more wary on the topic of humour: being conversational is one thing, but trying to be funny is a risk. "Documentation should not be about taking chances".
```

Comment:
I'm not keen on "on the side of formality" but don't have a great replacement.

Reply:
Maybe...?
- In favor of formality is that it can simplify localization...
- When it comes to formality, localization...

Reply:
great, thanks!